hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1503,7 +1503,7 @@ message TenantRevokeAdminRequest {
 }
 
 message GetS3VolumeRequest {
-  optional string accessID = 1;
+
```

Comment:
I believe I need to add this `accessID` field back if we piggyback the `accessID -> userPrincipal` request for KMS encryption. It's alright to remove it here in this patch.

Reply:
For that change, I believe you will just need to modify the response proto, not the request. The access ID should be determined by the OM's thread local s3 auth when the request is received.